Open Judge System/Web/OJS.Web/Areas/Administration/Views/Problems/Create.cshtml

```diff
@@ -252,13 +252,34 @@
                     </div>
                 </div>
                 <div class="col-xs-5">
-                    <div id="tests-file-button" class="btn btn-sm btn-primary pull-right">@Resource.Choose_file</div>
-                    <div class="hidden-file-upload">
-                        <input type="file" name="testArchive" id="tests-upload-input" />
+                    <div class="pull-right">
+                        @Html.EditorFor(m => m.Tests, new { onSelect = "testsValidation" })
```

Comment:
Change the name of `onSelect` to something more descriptive. Maybe 'onSelectCallback' or something like that.